app/src/main/res/layout/fragment_about_us.xml

```diff
@@ -124,14 +123,13 @@
 
                 </LinearLayout>
 
-            </android.support.v7.widget.CardView>
+            </LinearLayout>
 
             <!-- About Reporting Issues -->
-            <android.support.v7.widget.CardView
+            <LinearLayout
                 android:layout_width="match_parent"
-                app:cardCornerRadius="@dimen/margin_small"
                 android:layout_height="wrap_content"
-                android:layout_margin="@dimen/margin_small"
+                android:padding="5dp"
```

Comment:
Please use dimen resources instead of hardcoded dimens.

Reply:
@arundhati24 Sure 👍